lib/perl/Genome/Config.pm

```diff
@@ -0,0 +1,158 @@
+package Genome::Config;
+
+use strict;
+use warnings;
+
+use File::Find::Rule qw();
+use Genome::Carp qw(croakf);
+use Genome::ConfigSpec qw();
+use Path::Class qw();
+use YAML::Syck qw();
+
+=item get()
+
+C<get($key)> retrieves the configuration value specified by the input key.  It
+does a hierarchival search for the value,
+
+- environment variable
+- user config file
+- snapshot config file
+- global config files
```

Comment:
Yes, the global directories are specified by a colon-delimited environment variable, `XGENOME_CONFIG_DIRS`, defaulting to `/etc`.